Agilefreaks/FreaksCatalog-API
# [120] Add technologies to a freak

## Description
Create a Technology model and a database table named `technologies` for this model.
Create an association between Freak and Technology and a database table for this association named `freak_technologies` which helps us to make the association between these 2 models.
Also, the json data from technologies was removed,  now when we run the `freaks_spec` test a technology object is created in database and assigned for a freak.

## Related

[[120] Add Technologies to a Freak ](https://trello.com/c/uCnM0Nrv/120-add-technologies-to-a-freak)

## Steps to Test or Reproduce

The seeds.rb file will populate tables `technologies`, `freaks` and `freak_technologies` with some data. You can run next GraphQl query to get all freaks and see that data from technologies will be taken from database
```GraphQl

query Freaks($last: Int, $before: String) {
    freaks(last: $last, before: $before) {
        edges {
            node {
                projects {
                    technologies {
                        id
                        name
                        description
                    }
                }
                technologies {
                    id
                    name
                    description
                }
            }
        }
    }
}

```
